src/main/groovy/org/mockito/release/internal/gradle/util/PomCustomizer.groovy

```diff
@@ -84,13 +83,7 @@ class PomCustomizer {
 
             def contributorsNode = root.appendNode('contributors')
             def fileName = Contributors.getAllProjectContributorsFileName(project.getBuildDir())
-            AllProjectContributorsReader reader = AllProjectsContributorsProvider.allProjectContributorsReader()
-            def contributors = reader.loadAllContributors(fileName)
-            contributors.getAllContributors().each {
-                def c = contributorsNode.appendNode('contributor')
-                c.appendNode('name', it.name != "" ? it.name : it.login)
-                c.appendNode('url', it.profileUrl)
-            }
+            ContributorsToPom.include(contributorsNode, fileName, conf.team.contributors, conf.team.developers)
```

Comment:
I _really_ like the idea of pushing out this complexity to the separate Java class. It not only streamlines testing but also we have a chance to move some code from Groovy to Java!!!